package.json

```diff
@@ -38,15 +38,15 @@
     "prebuild": "rm -rf ./vendor"
   },
   "engines": {
-    "node": ">=12",
-    "npm": ">=6"
+    "node": ">=18",
+    "npm": ">=9"
```

Comment:
Node v16 is still compatible and gives 0 issues as well.

Reply:
Please refer to https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/345#issuecomment-1874827702.